src/components/ProfilePage.jsx

```diff
@@ -210,6 +211,8 @@ export class ProfilePage extends React.Component {
             </Col>
             <Col xs={{ order: 1 }} md={{ size: 8, order: 2 }} lg={{ size: 8, offset: 1 }} className="mt-4 mt-md-n5">
 
+              {this.props.requiresParentalConsent ? <AgeMessage accountURL="#account" /> : null}
```

Comment:
Shouldn't this link to the LMS account settings page until we create the new microfrontend version of that page?